app/assets/stylesheets/messages/base.css.scss

```diff
@@ -0,0 +1,17 @@
+.message {
+  text-align: left;
+  margin-top: 1em;
+  margin-left: 3em;
+
+  .message-header {
+
+    .button{
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Opening curly brace `{` should be preceded by one space